app/src/main/java/woowacourse/movie/presentation/view/main/home/MovieListFragment.kt

```diff
@@ -0,0 +1,27 @@
+package woowacourse.movie.presentation.view.main.home
+
+import android.os.Bundle
+import android.view.View
+import androidx.core.content.ContextCompat
+import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.RecyclerView
+import woowacourse.movie.R
+import woowacourse.movie.presentation.view.main.home.data.MovieData
+
+class MovieListFragment : Fragment(R.layout.fragment_movie_list) {
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        setListView(view)
+    }
+
+    private fun setListView(view: View) {
+        val listView = view.findViewById<RecyclerView>(R.id.rv_movie_list)
```

Comment:
```suggestion
        val recyclerView = view.findViewById<RecyclerView>(R.id.rv_movie_list)
```
recyclerView로 개선하였으니, recyclerView가 되어야하지 않을까요?